catroid/src/main/java/org/catrobat/catroid/stage/StageListener.java

```diff
@@ -185,25 +185,25 @@ public void create() {
 		scene = ProjectManager.getInstance().getCurrentlyPlayingScene();
 		pathForSceneScreenshot = PathBuilder.buildScenePath(project.getName(), scene.getName()) + "/";
 
-		virtualWidth = project.getXmlHeader().virtualScreenWidth;
-		virtualHeight = project.getXmlHeader().virtualScreenHeight;
-
-		virtualWidthHalf = virtualWidth / 2;
-		virtualHeightHalf = virtualHeight / 2;
-
-		camera = new OrthographicCamera();
-		viewPort = new ExtendViewport(virtualWidth, virtualHeight, camera);
-		if (batch == null) {
-			batch = new SpriteBatch();
+		if (stage == null) {
+			createNewStage();
+			if (DEBUG) {
```

Comment:
please remove DEBUG flag and all code associated. this is never used, and its back from 2012.